counter/example.go

```diff
@@ -0,0 +1,59 @@
+package counter
```

Comment:
Ah, right. Yepp, that makes sense! Would you add a comment to that effect in the file itself? I can see myself forgetting this and going through the exact same set of questions six months from now.